src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -610,6 +610,37 @@ public class C
             await VerifyCSharpAsync(source, shippedText, unshippedText);
         }
 
+        [Fact]
+        [WorkItem(3329, "https://github.com/dotnet/roslyn-analyzers/issues/3329")]
+        public async Task RemovedPrefixForNonRemovedApi()
+        {
+            var source = @"
+public class C
+{
+    public int Field;
+    public int Property { get; set; }
+    public void Method() { }
+}
+";
+
+            var shippedText = @"
+C
+C.C() -> void
+C.Field -> int
+C.Property.get -> int
+C.Property.set -> void
+C.Method() -> void
```

Comment:
Can you also add a test where `C.Method() -> void` is not present in the shipped text?